js/docEnums.js

```diff
@@ -1015,3 +1015,8 @@
  * @typedef {string} Enums.FileManagerItemViewMode
  * @enum {'details'|'thumbnails'}
  */
+
+/**
+ * @typedef {string} Enums.DiagramAutoLayout
```

Comment:
/cc @RomanTsukanov

Reply:
@Romzay Why do we need `Auto` in the name?